source/_components/binary_sensor.xiaomi.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title: "Xiaomi Binary Sensor"
+description: "Instructions how to setup the Xiaomi binary sensors within Home Assistant."
+date: 2017-07-21 16:34
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: xiaomi.png
+ha_category: Binary Sensor
+ha_release: "0.50"
+ha_iot_class: "Cloud Polling"
```

Comment:
It think "Cloud Polling" isn't correct. The communication between HA and the xiaomi gateway is multicast/udp at the local network. The multicast broadcast pushs new events. Polling is just used to fix some issues.